charts/backingservices/values.yaml

```diff
@@ -77,8 +77,14 @@ elasticsearch:
   # Shrink default JVM heap.
   esJavaOpts: "-Xmx1024m -Xms1024m"
   # Allocate smaller chunks of memory per pod.
-  protocol: http
   # This section specifies the elasticsearch cluster configuration for authentication and TLS.
+  # If `srs.srsStorage.tls.enabled: true` uncomment the blow parameter `protocol: https`
```

Comment:
If you previously set `srs.srsStorage.tls.enabled: true`, you must uncomment the line to use `protocol: https` parameter.